tests/CMakeLists.txt

```diff
@@ -66,31 +66,26 @@ add_subdirectory(unittest)
 # etc.
 add_subdirectory(integration)
 
-add_test(NAME integration_CoreExampleApp WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
-         COMMAND "$<TARGET_FILE:CoreExampleApp>"
+add_test(NAME integration_CoreExample WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/
```

Comment:
Do we still need lines above, from 20,
```
if(MSVC OR MSVC_IDE OR MINGW)
    # must find the RadiumExternal directory. This will be set by RadiumConfig.cmake after install
    #
    # TODO : find a way to that more efficiently
    if(DEFINED cpplocate_DIR)
        find_dependency(cpplocate REQUIRED NO_DEFAULT_PATH)
    endif()
    if(DEFINED glbinding_DIR)
        find_dependency(glbinding REQUIRED NO_DEFAULT_PATH)
    endif()
 # ...
```

Reply:
Not sure to understand why they are here ... So you can try without them and if it works ... remove them ^^

Reply:
well, some recent changes in CMakeLists.txt prevent to configure the project under windows. I'll fix it and push ASAP